src/App/root_reducer.js

```diff
@@ -0,0 +1,8 @@
+import {combineReducers} from 'redux';
+import login from '../Login/reducer';
```

Comment:
you don't really need that I guess, easy way to do is to set a alias in webpack file.  like 
```
{
    resolve: {
          root: ... 
    }
}
```